src/pages/en/guides/deploy/deno.md

```diff
@@ -21,6 +21,7 @@ Deploy an Astro site on Deno using the Deno adapter & [deployctl](https://github
     import deno from "@astrojs/deno";
 
     export default defineConfig({
+      output: 'server',
       adapter: deno()
```

Comment:
LGTM! Just pointing out that this deno one is the only one without a comma after the adapter. We could keep these all consistent.

```suggestion
      adapter: deno(),
```